package.json

```diff
@@ -20,7 +20,8 @@
     "ramda": "^0.26.1",
     "react-icons": "^3.9.0",
     "reakit": "^1.0.0-beta.14",
-    "styled-components": "^5.0.0"
+    "styled-components": "^5.0.0",
+    "yup": "^0.28.3"
```

Comment:
I don't think we'll make use `yup` in our own components. Can you make this a dev dependency?